### ISSUE TITLE
Git organization

### ISSUE THREAD

**Issue by distantnative** [getkart/kart#18]
Hey guys,
just wanted to quickly discuss how we'd want to organizing our work on github especially in terms of organization? Should we follow the "typical" git-flow scheme?

Branches:
master - last stable release
develop
feature/FEATURENAME 
hotfix/NAME
release/VERSION

I mean, we won't really have a release for a long time, I guess. Atm I keep just pushing to the master branch for the starterkit, but once we start working together on a repo, we should have a somehow consistent way, I think.

**Comment by plaidscreen** [comment]
I've been thinking about this too. (Which caused me to put the big warning on the READMEs)
Maybe we should move over to a "develop" branch, then after we have our first release, use the feature/* and hotfix/* schemes? The most current, stable release is always "master"? I'm really open to whatever.

**Comment by distantnative** [comment]
Switching over to a developer branch sounds good in general

**Comment by plaidscreen** [comment]
@distantnative Cool. I will switch over the kart and panel repos, if you want to do starterkit. The events repo may actually be ready for an unstable release. Want to do some live tests.

**Comment by benseitz** [comment]
I really like feature branches, that merge into developer. Master for last stable release will be required when we actually have releases AND stable releases :D

**Comment by plaidscreen** [comment]
After the first public release, I like feature branches. When I tried to do feature branches pre-launch, I just ended up with 200 branches that ended up getting deleted later anyway. :smile: But maybe that's just another good incentive to keep Kart tight and lean on the core feature-set?

**Comment by distantnative** [comment]
The problem could be if we don't use other branches than the develop branch and pull request, but just push to the develop branch, that we run into conflicts, working on the same branch simultaneously/overlapping. Right?

**Comment by benseitz** [comment]
Imho, when having multiple contributors feature branches come in handy :)

**Comment by plaidscreen** [comment]
@distantnative Yep, that happens. I haven't shared a Git repo for a few years, forgot about merge conflicts. I think you're both right, we should split our projects off into feature branches, it'll also give a quick glimpse of what feature is being worked on at any given moment.

**Comment by benseitz** [comment]
:+1:

**Comment by distantnative** [comment]
:+1: the starterkit will be rather idle-ing for the moment as it makes more sense to get it going, when Kart with its methods starts to take shape
